src/figuresClasses.ts

```diff
@@ -1,11 +1,71 @@
-export interface Figure {}
+export interface Figure {
+  shape: string;
```

Comment:
Shape should be an enum with predefined values, so I know I did something wrong when I passed `horse` there.